spec/data-model/README.md

```diff
@@ -214,11 +220,43 @@ that the implementation attaches to that _annotation_.
 ```ts
 interface UnsupportedAnnotation {
   type: "unsupported-annotation";
-  sigil: "!" | "@" | "#" | "%" | "^" | "&" | "*" | "<" | ">" | "/" | "?" | "~";
+  sigil: "!" | "@" | "%" | "^" | "&" | "*" | "+" | "<" | ">" | "?" | "~";
   source: string;
 }
 ```
 
+## Markup
+
+A `Markup` object is either `MarkupOpen`, `MarkupStandalone`, or `MarkupClose`,
+which are differentiated by `kind`.
```

Comment:
Should we call out that open/standalone/close carries no semantic meaning, unless assigned by the implementation?

Reply:
I don't think that necessarily belongs here in the data model, given that it's an optional part of the spec?

Reply:
I think we should call it out, but agree that it doesn't belong here.